softdevice_controller/CHANGELOG.rst

```diff
@@ -27,12 +30,16 @@ Changes
 Bug fixes
 =========
 
+* Fixed an issue where a slave connection could disconnect prematurely if there were scheduling conflicts with other roles (DRGN-15469).
 * Fixed an issue where the channel map provided by the LE Host Set Channel Classification HCI command was not applied on the secondary advertising channels (DRGN-13594).
 * The SoftDevice Controller can now be qualified on nRF52832 (DRGN-15382).
 * Fixed an issue where setting a legacy advertiser's scan response data using extended advertising HCI commands corrupted the advertising data (DRGN-15465).
 * Fixed an issue where, in rare cases, an assert could occur when receiving a packet as a slave.
   This could only occur after performing a data length procedure on Coded PHY (DRGN-15251).
 * Fixed an issue where "HCI Read RSSI" would always return a Command Disallowed (0x0C) error code (DRGN-15310).
+* Fixed an issue where setting radio output power using the Vendor specific HCI command: Zephyr Write TX Power Level returned "Unsupported Feature or Parameter value (0x11).
```

Comment:
```suggestion
* Fixed an issue where setting radio output power using the vendor-specific HCI command Zephyr Write TX Power Level returned "Unsupported Feature or Parameter value (0x11)".
```

Reply:
We use "Vendor specific" in many other places in this document. Should we update all occurrences to use "vendor-specific"?

Reply:
Yes, please (unless it's part of the actual command name).